velox/exec/HashBuild.cpp

```diff
@@ -538,6 +502,8 @@ bool HashBuild::reserveMemory(const RowVectorPtr& input) {
 void HashBuild::spillInput(const RowVectorPtr& input) {
   VELOX_CHECK_EQ(input->size(), activeRows_.size());
 
+  // Q(jtan6): Why do we have !spiller_->isAnySpilled() condition here? If
```

Comment:
Yes, if nothing has been spilled, then we don't spill the input. Can you remove the question note? Thanks!